src/lua/luaTask.c

```diff
@@ -241,6 +242,15 @@ static void lua_task(void *params)
 
                 /* If its a known unrecoverable, fail fast */
                 switch (rc) {
+		case LUA_NO_PERIODIC_FUNCTION:
```

Comment:
Yes.  This should do that as that happens when the script is loaded, not when the periodic function is run.